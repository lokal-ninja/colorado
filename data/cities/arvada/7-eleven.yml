id: 743748900
type: way
latitude: 39.85667
longitude: -105.16455
postcode: "80005"
city: Arvada
street: "Indiana Street"
housenumber: 8790
phone: "+1-720-357-1618"
opening_hours: "24/7"
website: "https://www.7-eleven.com/locations/co/arvada/8790-indiana-st-38226"
