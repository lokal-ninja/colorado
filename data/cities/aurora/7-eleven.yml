id: 2906579957
type: node
latitude: 39.71075
longitude: -104.86326
postcode: "80012"
city: Aurora
street: "East Alameda Avenue"
housenumber: 10722
phone: "303-341-0346"
opening_hours: ""
website: ""