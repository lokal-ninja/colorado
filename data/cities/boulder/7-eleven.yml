id: 2203468008
type: node
latitude: 40.00716
longitude: -105.27643
postcode: ""
city: Boulder
street: "13th Street"
housenumber: 1091
phone: ""
opening_hours: "24/7"
website: ""
