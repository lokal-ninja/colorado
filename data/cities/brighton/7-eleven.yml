id: 2645055845
type: node
latitude: 39.97159
longitude: -104.77298
postcode: "80601"
city: Brighton
street: "Bromley Lane"
housenumber: 3876
phone: ""
opening_hours: ""
website: ""
