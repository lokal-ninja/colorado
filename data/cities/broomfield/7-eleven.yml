id: 297611075
type: way
latitude: 39.91473
longitude: -105.03962
postcode: "80020"
city: Broomfield
street: "West 120th Avenue"
housenumber: 4175
phone: "+1-303-460-8470"
opening_hours: "24/7"
website: "https://www.7-eleven.com/locations/co/broomfield/4175-w-120th-39742"