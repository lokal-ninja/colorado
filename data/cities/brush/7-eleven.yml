id: 3782927038
type: node
latitude: 40.25498
longitude: -103.62688
postcode: 80723-2012
city: Brush
street: EDISON
housenumber: "620"
phone: +1 970-842-3123
opening_hours: ""
website: https://www.7-eleven.com/locations/co/brush/620-edison-15955
