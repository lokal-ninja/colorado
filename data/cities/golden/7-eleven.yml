id: 658887087
type: node
latitude: 39.7682
longitude: -105.23514
postcode: "80403"
city: Golden
street: "North Washington Avenue"
housenumber: 399
phone: "+1-303-216-9163"
opening_hours: "24/7"
website: "https://www.7-eleven.com/locations/co/golden/399-n.-washington-avenue-33481"
